src/graph/KokkosGraph_GraphColorHandle.hpp

```diff
@@ -620,6 +620,7 @@ class GraphColoringHandle
       this->eb_num_initial_colors = 1;
       break;
     default:
+      std::cout << "col_algo:" << col_algo << std::endl;
```

Comment:
@mndevec Is this debugging output or intentional?

Reply:
ops :)

Reply:
I pushed to delete this.